lib/utils.pm

```diff
@@ -1685,4 +1686,17 @@ sub assert_file_content {
     my ($param, $value) = @_;
     assert_script_run("cat $param | grep $value");
 }
+
+=head2  get_package_for_install
+        Get package name from PACKAGETOINSTALL
+        if variable is not defined :
+        - use 'x3270' for sle
+        - or 'xdelta3'  for openSUSE
+=cut
+sub get_package_for_install {
+    my $default_pkgname = ' ';
+    $default_pkgname = 'x3270'   if check_var('DISTRI', 'sle');
+    $default_pkgname = 'xdelta3' if check_var('DISTRI', 'opensuse');
+    return get_var('PACKAGETOINSTALL', $default_pkgname);
+}
```

Comment:
```
_______________________________________________________________________________________________________________________________________________________________________________________________➜  ~ cat /etc/os-release
NAME="openSUSE Tumbleweed"
# VERSION="20200409"
ID="opensuse-tumbleweed"
ID_LIKE="opensuse suse"
VERSION_ID="20200409"
PRETTY_NAME="openSUSE Tumbleweed"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:tumbleweed:20200409"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
LOGO="distributor-logo"
_______________________________________________________________________________________________________________________________________________________________________________________________➜  ~ zypper se -sv x3270
Loading repository data...
Reading installed packages...

S  | Name              | Type       | Version | Arch   | Repository               
---+-------------------+------------+---------+--------+--------------------------
i+ | x3270             | package    | 3.5-3.1 | x86_64 | (System Packages)        
    name: x3270
v  | x3270             | package    | 3.6-1.1 | x86_64 | Main Repository (OSS)    
    name: x3270
v  | x3270             | package    | 3.6-1.1 | i586   | Main Repository (OSS)    
    name: x3270
   | x3270             | srcpackage | 3.6-1.1 | noarch | Main Repository (Sources)
    name: x3270
   | x3270-debuginfo   | package    | 3.6-1.1 | x86_64 | Main Repository (DEBUG)  
    name: x3270-debuginfo
   | x3270-debuginfo   | package    | 3.6-1.1 | i586   | Main Repository (DEBUG)  
    name: x3270-debuginfo
   | x3270-debugsource | package    | 3.6-1.1 | x86_64 | Main Repository (DEBUG)  
    name: x3270-debugsource
   | x3270-debugsource | package    | 3.6-1.1 | i586   | Main Repository (DEBUG)  
    name: x3270-debugsource
```

sub get_package_for_install {
    return get_var('PACKAGETOINSTALL',  'x3270');
}

Reply:
In fact, I would rather avoid anything which needs Windows Manager of any kind.